vmware_dvs/utils/dvs_util.py

```diff
@@ -534,12 +552,15 @@ def blocked(self, value):
 
 def create_network_map_from_config(config):
     """Creates physical network to dvs map from config"""
+    # It needs to increase connection pool for multiple
+    # requests
     connection = api.VMwareAPISession(
         config.vsphere_hostname,
         config.vsphere_login,
         config.vsphere_password,
         config.api_retry_count,
-        config.task_poll_interval)
+        config.task_poll_interval,
+        pool_size=100)
```

Comment:
Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts)